Core/src/org/sleuthkit/autopsy/keywordsearchservice/KeywordSearchService.java

```diff
@@ -48,13 +48,19 @@
      * all of its attributes.
      *
      * @param artifact The artifact to index.
+     * 
+     * @deprecated postArtifact org.sleuthkit.autopsy.datamodel.Blackboard will
+     * take care of this functionality at ingest
```

Comment:
This seems a bit choppy: method name, class name. Perhaps simply say: "Call org.sleuthkit.autopsy.datamodel.Blackboard.postArtifact instead."